docs/api.md

```diff
@@ -278,6 +279,13 @@ Possible error code responses. New error codes may be added in future releases.
 | `unsupported_test_type` | 412         | No    | The code may be valid, but represents a test type the client cannot process. User may need to upgrade software. |
 |                         | 500         | Yes   | Internal processing error, may be successful on retry.                                                          |
 
+### Client provided UUID to prevent duplicate SMS
+
+Every response includes `uuid` to track the status of an issued code. Optionally `IssueCodeRequest` may also take in a `uuid` from the client. This can be useful when implementing retry logic to ensure the same request does not send more than one SMS to the same patient. Once successful, subsequent requests with the same `uuid` will give status `409` `uuid_already_exists`.
```

Comment:
we also want to talk about how this can be used from say a lab / test provider to send a code and send the UUID along to the state to ensure a code isn't issued again.

Reply:
I'm not sure I follow. Would both the lab and the state be sending codes?